### PR TITLE
CloudEvents Rust tabbed doc with kn 

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Like https://github.com/knative/docs/pull/2805 but for the cloudevents-rust sample

### CODE REVIEW

File: docs/serving/samples/cloudevents/cloudevents-rust/index.md [hunk 1]
```diff
@@ -5,4 +5,163 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```
Comment (abrennan89) [inline docs/serving/samples/cloudevents/cloudevents-rust/index.md:8]: cc @RichieEscarez 
Do we need to use the index+README for some reason or is it OK to just add the content to index like this?

Comment (slinkydeveloper) [reply docs/serving/samples/cloudevents/cloudevents-rust/index.md:8]: @mattmoor removed it here https://github.com/knative/docs/pull/2805

Comment (mattmoor) [reply docs/serving/samples/cloudevents/cloudevents-rust/index.md:8]: @abrennan89 I think Github prefers README, but the `readfile` shortcode doesn't work with nested shortcodes IIUC

**Comment by slinkydeveloper** [comment]
@mattmoor ping

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**